dashboard/src/app/navbar/navbar.html

```diff
@@ -32,15 +32,15 @@
             <md-list-item flex class="navbar-subsection-item">
               <md-button nav-bar-selected flex che-reload-href href="{{navbarController.menuItemUrl.dashboard}}" layout-align="left"
                          target="_self">
-                <div class="navbar-item" layout="row" layout-align="start center">
+                <div class="navbar-item" layout="row" layout-align="start center" id="dashboard-item">
```

Comment:
would it be safer if ids are prefixed ?
like navbar-dashboard-item

it would avoid any collision between ids